src/lambda/templates/lambdaTemplates.ts

```diff
@@ -20,7 +20,8 @@ export class LambdaTemplates {
         <h3>
             Select a file to use as payload:
         </h3>
-        <input type="file" @change="processFile($event)">
+        <button v-on:click="promptForFileLocation">Choose File</button>
+        <span>  {{ selectedFile }}</span>
```

Comment:
- should this be in a `<div>` to avoid wrapping?
- how does it look if the selectedFile name is very long?

```suggestion
        <div style="text-overflow: ellipsis;">
        <button v-on:click="promptForFileLocation">Choose File</button>
        <span>  {{ selectedFile }}</span>
        </div>
```

Reply:
A <div> will always put it on a new line.  I think I like it next to the button.

I can add `white-space: nowrap` to the CSS file to prevent word wrapping if that's the desired behavior.

Reply:
> A `<div>` will always put it on a new line. I think I like it next to the button.

 the `<div>` in the suggestion above wraps both, with the intention of keeping the filename next to the button. adding a wrap style to the `<span>` won't accomplish that.

Reply:
Committed your suggestion